### PR TITLE
Fix and improve operator docker guide

### PR DESCRIPTION
**Pull request by EmilFattakhov**
This PR fixed the docker instructions for running an operator, and running an operator-validator node. 
Will do some testing to see if there's any flags that needs to be set / removed.

### CODE REVIEW

File: docs/farming-&-staking/staking/operators.mdx [hunk 1]
```diff
@@ -262,22 +262,27 @@ services:
...
       "--listen-addr", "/ip4/0.0.0.0/tcp/40333",
       "--rpc-cors", "all",
       "--rpc-methods", "unsafe",
       "--rpc-external",
+      # Replace INSERT_YOUR_ID with your node ID (will be shown in telemetry)
+      "--name", "INSERT_YOUR_ID",
+      "--",
+      # Replace INSERT_YOUR_DOMAIN_ID with domain ID you want to be operator on
+      "--domain-id", "INSERT_YOUR_DOMAIN_ID",
+      "--chain", "gemini-3g",
```
Comment (jim-counter) [inline docs/farming-&-staking/staking/operators.mdx:279]: Pretty sure we don't need the chain on the operator side. The chain comes from the consensus side.

File: docs/farming-&-staking/staking/operators.mdx [hunk 2]
```diff
@@ -262,22 +262,27 @@ services:
...
-      "--operator-id", "your_operator_id",
-      "--bootnodes", "/ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp",
-      "--bootnodes", "/ip4/140.82.45.36/tcp/40333/p2p/12D3KooWJxB2PVzyNqZywoCWAMLGhKr2LCbFMXbAPzUPQZon7yt1",
-      "--bootnodes", "/ip4/167.179.116.135/tcp/40333/p2p/12D3KooWHsRae3ip2uQAUPtpMcStHGGw777SJT3nu9maFmtsGry7",
+      "--base-path", "/var/subspace",
+      "--port", "30333",
+      "--dsn-listen-on", "/ip4/0.0.0.0/udp/30433/quic-v1",
+      "--dsn-listen-on", "/ip4/0.0.0.0/tcp/30433",
       "--listen-addr", "/ip4/0.0.0.0/tcp/40333",
       "--rpc-cors", "all",
```
Comment (jim-counter) [inline docs/farming-&-staking/staking/operators.mdx:271]: We only need the `rpc-` parameters if we are connecting a farmer. And this is the pure operator example

File: docs/farming-&-staking/staking/operators.mdx [hunk 3]
```diff
@@ -262,22 +262,27 @@ services:
...
-      "--domain-id", "your_domain_id",
-      "--operator-id", "your_operator_id",
-      "--bootnodes", "/ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp",
-      "--bootnodes", "/ip4/140.82.45.36/tcp/40333/p2p/12D3KooWJxB2PVzyNqZywoCWAMLGhKr2LCbFMXbAPzUPQZon7yt1",
-      "--bootnodes", "/ip4/167.179.116.135/tcp/40333/p2p/12D3KooWHsRae3ip2uQAUPtpMcStHGGw777SJT3nu9maFmtsGry7",
+      "--base-path", "/var/subspace",
+      "--port", "30333",
+      "--dsn-listen-on", "/ip4/0.0.0.0/udp/30433/quic-v1",
+      "--dsn-listen-on", "/ip4/0.0.0.0/tcp/30433",
       "--listen-addr", "/ip4/0.0.0.0/tcp/40333",
```
Comment (jim-counter) [inline docs/farming-&-staking/staking/operators.mdx:270]: This `listen-addr` should be on the operator side.

File: docs/farming-&-staking/staking/operators.mdx [hunk 4]
```diff
@@ -262,22 +262,27 @@ services:
       - "0.0.0.0:30433:30433/tcp"
```
Comment (jim-counter) [inline docs/farming-&-staking/staking/operators.mdx:262]: We need to add 40333 to this list.

File: docs/farming-&-staking/staking/operators.mdx [hunk 5]
```diff
@@ -262,22 +262,27 @@ services:
...
+      "--",
+      # Replace INSERT_YOUR_DOMAIN_ID with domain ID you want to be operator on
+      "--domain-id", "INSERT_YOUR_DOMAIN_ID",
+      "--chain", "gemini-3g",
+      # Replace INSERT_YOUR_OPERATOR_ID with your operator ID
+      "--operator-id", "INSERT_YOUR_OPERATOR_ID",
+      "--bootnodes", "/ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp",
+      "--bootnodes", "/ip4/140.82.45.36/tcp/40333/p2p/12D3KooWJxB2PVzyNqZywoCWAMLGhKr2LCbFMXbAPzUPQZon7yt1",
+      "--bootnodes", "/ip4/167.179.116.135/tcp/40333/p2p/12D3KooWHsRae3ip2uQAUPtpMcStHGGw777SJT3nu9maFmtsGry7",
       "--keystore-path", "/keystore"
```
Comment (jim-counter) [inline docs/farming-&-staking/staking/operators.mdx:285]: We will need to mount this folder in the `volumes` section. Something like `- /path/to/keystore:/keystore:ro`.

**Review by jim-counter — CHANGES_REQUESTED** [review]
Some changes required here then I'll re-review. Thanks Emil.

File: docs/farming-&-staking/staking/operators.mdx (hunk 1 above)
Comment (EmilFattakhov) [reply docs/farming-&-staking/staking/operators.mdx:279]: Makes sense!

File: docs/farming-&-staking/staking/operators.mdx (hunk 2 above)
Comment (EmilFattakhov) [reply docs/farming-&-staking/staking/operators.mdx:271]: Left in the farmer example, removed from the pure operator.

**Comment by EmilFattakhov** [comment]
Thanks @jim-counter, please have another look when you get a chance!

**Comment by EmilFattakhov** [comment]
@vedhavyas I'd appreciate if you could have a look to see if there's anything obvious missing, thanks!

**Pull request merged**